lms/static/scripts/frontend_apps/components/ContentSelector.js

```diff
@@ -203,8 +203,8 @@ export default function ContentSelector({
     setLoadingIndicatorVisible(true);
     try {
       const picker = /** @type {OneDrivePickerClient} */ (oneDrivePicker);
-      const { url } = await picker.showPicker();
-      onSelectContent({ type: 'url', url });
+      const { name, url } = await picker.showPicker();
+      onSelectContent({ type: 'url', name, url });
```

Comment:
Same ordering as the one previous.
```suggestion
      onSelectContent({ name, type: 'url', url });
```